### PR TITLE
add swish component example

### PR DESCRIPTION
**Pull request by shieldo**
Adding some code for the Swish component - caveat: some of the points in the app where NL is hard-coded as the country code and EUR as the currency need to be changed to SE and SEK respectively in order to work.

### CODE REVIEW

File: src/swish/swish.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        onError: error => {
+            console.log(error);
+        },
+    });
+
+    const swishData = {
+        paymentMethod: {
+            type: 'swish'
+        },
+        returnUrl: '.'
```
Comment (marcperez) [inline src/swish/swish.js]:
```suggestion
        countryCode: 'SE',
        amount: { currency: 'SEK', value: 1000 }
```

We are already adding a `returnUrl` in the `utils.js` file. For this sample code purpose we can override the `countryCode` and `amount` from the frontend (of course that should never be done in production).

File: src/swish/swish.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    const checkout = new AdyenCheckout({
+        originKey, // Mandatory. originKey from Costumer Area
+        environment: 'test',
+        amount: { currency: 'EUR', value: 1000 },
+        onAdditionalDetails: result => {
+            console.log(result);
+        },
+        onError: error => {
+            console.log(error);
+        },
```
Comment (marcperez) [inline src/swish/swish.js]:
```suggestion
        }
```

File: src/swish/swish.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        paymentMethod: {
+            type: 'swish'
+        },
+        returnUrl: '.'
+    };
+
+    makePayment(swishData).then(response => {
+        checkout.createFromAction(response.action).mount('#swish-container');
+    });
+});
```
Comment (marcperez) [inline src/swish/swish.js]:
```suggestion
});

```

**Review by marcperez — CHANGES_REQUESTED** [review]
Hi @shieldo,

Thanks again for your PRs! I added a couple of comments.

Best,
Marc

**Comment by shieldo** [comment]
Thanks for the feedback @marcperez - have made tweaks as suggested.

File: src/swish/swish.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    });
+
+    const swishData = {
+        paymentMethod: {
+            type: 'swish'
+        },
+        countryCode: 'SE',
+        amount: {
+            currency: 'SEK',
+            amount: 10000
```
Comment (marcperez) [inline src/swish/swish.js]:
```suggestion
            value: 10000
```

**Review by marcperez — CHANGES_REQUESTED** [review]
Hi @shieldo Just one small typo and we can merge it. Thanks again! 👍

**Pull request merged**